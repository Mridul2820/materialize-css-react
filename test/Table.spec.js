import React from 'react';
import { render } from '@testing-library/react';
import Table from '../src/Table';

describe('<Table />', () => {
  test('renders', () => {
    const { container } = render(
      <Table centered hoverable responsive striped>
        <tbody>
          <tr>
            <td>foo</td>
            <td>bar</td>
          </tr>
        </tbody>
      </Table>
    );

    expect(container).toMatchSnapshot();
  });
});
