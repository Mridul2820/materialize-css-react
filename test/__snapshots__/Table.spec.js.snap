// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table /> renders 1`] = `
<div>
  <table
    class="centered highlight responsive-table striped"
  >
    <tbody>
      <tr>
        <td>
          foo
        </td>
        <td>
          bar
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
